### PR TITLE
pkg/etcd: return revision when create key

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 
return revision when creating a key. used for dm

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: pkg/etcd/etcd.go [hunk 1]
```diff
@@ -70,22 +70,27 @@ func (e *Client) Close() error {
...
+		return 0, errors.AlreadyExistsf("key %s in etcd", key)
 	}
 
-	return nil
+	if txnResp.Header != nil {
+		return txnResp.Header.Revision, nil
+	}
+
+	// impossible to happen
+	return -1, nil
```
Comment (csuzhangxc) [inline pkg/etcd/etcd.go]: How about returning a new error?

**Review by csuzhangxc — COMMENTED** [review]
rest LGTM

File: pkg/etcd/etcd.go (hunk 1 above)
Comment (WangXiangUSTC) [reply pkg/etcd/etcd.go]: updated https://github.com/pingcap/docs-cn/pull/2037

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Review by lichunzhu — APPROVED** [review]
LGTM

**Pull request merged**